### ISSUE TITLE
Feature an interesting image on the front page of the website (the one with the empty tab)

### ISSUE THREAD

**Issue by UnboltedWork** [MrAlex94/Waterfox#2688]
**Is your feature request related to a problem? Please describe.**
It looks plain empty, and not professional in my opinion, as that amount of space is being wasted.

**Describe the solution you'd like**
Simply add a great looking image, some gradient, swirl, great colors or simple colors. It really would make a difference.

**Describe alternatives you've considered**
Remove or shrink the empty tab, because I just can't really see anything special about it.

**Additional context**
This space could really be used for something.
<img width="1437" alt="Screen Shot 2022-04-25 at 8 18 25 pm" src="https://user-images.githubusercontent.com/59249514/165070303-804e15c8-2b89-4753-bbc5-9a98ae66aef9.png">

**Comment by taciturnaxolotl** [comment]
I would agree. This would make it look much smarter and not like its almost broken.

**Comment by Zulgrib** [comment]
Please don't bloat the new tab page, it needed to open quickly
